File: src/JavaScript/easy/string/problem1309.js
/**
 * @param {string} sentence
 * @return {string}
 */
var toGoatLatin = function (sentence) {
  let text = "";
  for (let i = 0; i < sentence.split(" ").length; i++) {
    if (["a", "e", "i", "o", "u"].includes(sentence.split(" ")[i][0].toLowerCase())) {
      let b = sentence.split(" ")[i] + "ma" + "a".repeat(i + 1);
      text += text.length == 0 ? b : " " + b;
    } else {
      let a = sentence.split(" ")[i].slice(1, sentence.split(" ")[i].length) +
      sentence.split(" ")[i][0] +"ma" + "a".repeat(i + 1);
      text += text.length == 0 ? a : " " + a;
    }
  }
  return text;
};

let sentence = "I speak Goat Latin";

toGoatLatin(sentence);

File: src/Python/easy/string/problem1309.py
class Solution:
    def toGoatLatin(self, sentence: str) -> str:
        text = ''
        for i in range(len(sentence.split(" "))):
            if sentence.split(" ")[i][0].lower() in ["a", "e", "i", "o", "u"]:
                a = sentence.split(" ")[i] + "ma" + "a" * (i + 1)
                text += a if len(text) == 0 else " " + a
            else:
                b = sentence.split(" ")[i][1:len(sentence.split(" ")[i])] + sentence.split(" ")[i][0] +"ma" + "a" * (i + 1)
                text += b if len(text) == 0 else " " + b
        return text


sentence = "I speak Goat Latin"

solution = Solution()
solution.toGoatLatin(sentence)